src/components/HistoryPanel.tsx

```diff
@@ -38,31 +39,46 @@ const cardVariant = {
   },
 };
 
-export default function VotePanel(_props: Props) {
+export default function HistoryPanel(_props: Props) {
   const { status: instanceStatus, data: instance } = usePublishedDatum(
     'agoricNames.instance',
   );
-  const { status: qStatus, data: questions } = usePublishedHistory(
-    'committees.Economic_Committee.latestQuestion',
-  );
-  const { status: aStatus, data: outcomes } = usePublishedHistory(
-    'committees.Economic_Committee.latestOutcome',
-  );
+  const {
+    status: qStatus,
+    data: questions,
+    fetchNextPage: fetchNextQs,
+  } = usePublishedHistory('committees.Economic_Committee.latestQuestion', 5);
+  const {
+    status: aStatus,
+    data: outcomes,
+    fetchNextPage: fetchNextAs,
+  } = usePublishedHistory('committees.Economic_Committee.latestOutcome', 5);
+
+  const loaderRef = useRef(null);
+  useEffect(() => {
+    const observer = new IntersectionObserver(entries => {
+      const target = entries[0];
+      if (target.isIntersecting) {
+        fetchNextQs();
+        fetchNextAs();
```

Comment:
"As" is not super obvious, especially since the vpath is "outcomes". consider spelling out the words. if it's too many characters, "Next" is somewhat implied
```suggestion
        fetchQuestions();
        fetchOutcomes();
```